server/src/test/java/org/elasticsearch/index/engine/LuceneChangesSnapshotTests.java

```diff
@@ -261,6 +271,34 @@ public void run() {
         }
     }
 
+    public void testSearchMultipleBatches() throws Exception {
```

Comment:
sine we now randomize the batching in tests, i don't think we need this dedicated test any more?